index.html

```diff
@@ -414,7 +414,7 @@ <h4>Simple Forms</h4>
         </footer>
 
 
-        <script src="./releases/alpha-.10/deer.js" type="module"></script>
+        <script src="./js/deer.js" type="module"></script>
```

Comment:
I'll put this back before merging. This should be our most recent release and the text version needs to change too.

Reply:
Yeah we did a fresh release and i was wondering if you wanted to up the version.  We should do that today.